packages/terra-search-field/src/SearchField.jsx

```diff
@@ -172,6 +174,9 @@ class SearchField extends React.Component {
 
   handleKeyDown(event) {
     if (event.nativeEvent.keyCode === KeyCode.KEY_RETURN) {
+      // set focus to search button to hide keyboard on mobile devices
+      this.searchBtnRef.focus();
```

Comment:
Can we add some tests to cover this change?

Reply:
A WDIO test to ensure button is focused on return key press !!

Reply:
Added test here:
f4db4098cfb69d72c9f50e870151647eae859479